src/components/common/steps_modal/buy_sell_collectible_step.tsx

```diff
@@ -101,6 +113,11 @@ class BuySellCollectibleStep extends React.Component<Props, State> {
                 onLoading();
                 await this.props.submitCollectibleOrder(signedOrder);
                 onDone();
+
+                // Go to the collectible's profile
+                if (!currentRoutePath.includes(`collectible/${collectible.tokenId}`)) {
```

Comment:
I think we need to stop doing this. We are hardcoding partial URLs everywhere and relaying a lot on this pattern of `currentRoutePath.includes`. It smells bad to me.

I don't know how is this supposed to be done the right way in `react-router`, but there must be a better way.

Reply:
Maybe we can dispatch [`goToIndividualCollectible`](https://github.com/0xProject/0x-launch-kit-frontend/blob/development/src/store/router/actions.ts#L70) and move this verification there?

To me, it seems that the condition check on the redirect is not a responsability of this component/file.

Reply:
> Maybe we can dispatch [`goToIndividualCollectible`](https://github.com/0xProject/0x-launch-kit-frontend/blob/development/src/store/router/actions.ts#L70) and move this verification there?
> 
> To me, it seems that the condition check on the redirect is not a responsability of this component/file.

@unjapones  Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/502/commits/bc6b4686fcbe044c926f617111d27c157517a598)